Frends.AzureBlobStorage.ReadBlob/Frends.AzureBlobStorage.ReadBlob/ReadBlob.cs

```diff
@@ -0,0 +1,74 @@
+﻿using System;
+using System.ComponentModel;
+using System.Threading;
+using System.Text;
+using Azure.Storage.Blobs;
+using Azure;
+using Frends.AzureBlobStorage.ReadBlob.Definitions;
+
+namespace Frends.AzureBlobStorage.ReadBlob
+{
+    public class AzureBlobStorage
+    {
+        /// <summary>
+        /// Read a single file to Azure blob storage using connection string or SAS Token.
+        /// [Documentation](https://tasks.frends.com/tasks#frends-tasks/Frends.AzureBlobStorage.ReadBlob)
+        /// <param name="source">Information about which Blob to read.</param>
+        /// <returns>Object { string content }</returns>
+        /// </summary>
+
+        public static Result ReadBlob([PropertyTab] Source source, CancellationToken cancellationToken)
+        {
+            var data = ReadBlobContent(source, cancellationToken);
+            return new Result(data.ToString());
+        }
+
+        private static string ReadBlobContent(Source source, CancellationToken cancellationToken)
+        {
+            BlobClient blob = null;
+            var uri = $"{source.Uri}/{source.ContainerName}/{source.BlobName}?";
+
+            switch (source.AuthenticationMethod)
+            {
+                case AuthenticationMethod.Connectionstring:
+                    if (string.IsNullOrEmpty(source.ConnectionString))
+                        throw new Exception("Connection string required.");
+                    blob = new BlobClient(source.ConnectionString, source.ContainerName, source.BlobName);
+                    break;
+
+                case AuthenticationMethod.Sastoken:
+                    if (string.IsNullOrEmpty(source.SasToken) || string.IsNullOrEmpty(source.Uri))
+                        throw new Exception("SAS Token and URI required.");
+                    blob = new BlobClient(new Uri(uri), new AzureSasCredential(source.SasToken));
+                    break;
+
+                default:
```

Comment:
I think default can be removed, since there are only two options and user has to select one of them.